packages/runtime/test/integration/pool-transaction-fee-test.ts

```diff
@@ -3,28 +3,40 @@ import { assert } from "chai";
 
 import { RUNTIME_ERRORS } from "../../src/errors/errors-list";
 import { AccountStore, Runtime } from "../../src/index";
-import { ALGORAND_ACCOUNT_MIN_BALANCE } from "../../src/lib/constants";
 import { useFixture } from "../helpers/integration";
 import { expectRuntimeError } from "../helpers/runtime-errors";
 
 describe("Pooled Transaction Fees Test", function () {
-	useFixture("app-update");
-	const minBalance = ALGORAND_ACCOUNT_MIN_BALANCE;
-	let john = new AccountStore(1e30);
-	let alice = new AccountStore(minBalance);
-	let bob = new AccountStore(minBalance);
-
+	useFixture("deploy-asa");
+	const initialBalance = 1e30;
+	let john: AccountStore;
+	let bob: AccountStore;
+	let elon: AccountStore;
+	let alice: AccountStore;
+	let assetId: number;
 	let runtime: Runtime;
 
 	this.beforeEach(async function () {
-		runtime = new Runtime([john, alice, bob]); // setup test
+		john = new AccountStore(initialBalance, "john");
```

Comment:
It has to be unfunded

Reply:
Ok. Done. Working with unfunded account.